### ISSUE TITLE
'Server' object has no attribute 'services'

### ISSUE THREAD

**Issue by Clives-online** [Esri/arcgis-python-api#1043]
**Describe the bug**

**To Reproduce**
```python
<gis = GIS(portalURL, username, password)
##print('Connected to {} as {}'.format(gis_name.properties.portalHostname, gis_name.users.me.username))

print("Logged IN")
#print(gis.admin.servers.validate())  ### true
gis_server = gis.admin.servers.list()
print(gis_server)
server1 = gis_server[0]
services = server1.services.list()
print(services)>
```
error:
```python
<Traceback (most recent call last):
  File "C:/Users/tmpgcesn/PycharmProjects/TEST/venv/Test/LOGS_Internal.py", line 31, in <module>
    services = server1.services.list()
  File "C:\Program Files\ArcGIS\Pro\bin\Python\envs\arcgispro-py3\lib\site-packages\arcgis\gis\server\_common\_base.py", line 86, in __getattr__
    raise AttributeError("'%s' object has no attribute '%s'" % (type(self).__name__, name))
AttributeError: 'Server' object has no attribute 'services'

Process finished with exit code 1>
```
**Screenshots**

**Expected behavior**
I followed the ESRI instruction for the arcgis module, expected a list of Services

**Platform (please complete the following information):**
 - OS: [Windows 10]
 - Browser [Chrome, Edge]
 - Python API Version [1.8.3] 

**Additional context**
I have used Pycharm and Jypter Notebooks using Python 3.7.4, and get the same results.
I am following the instructions from: https://developers.arcgis.com/python/guide/managing-your-gis-servers/

**Comment by achapkowski** [comment]
@Clives-online 1.8.3 is fairly old, please try updating the library.

**Comment by jyaistMap** [comment]
* cannot reproduce:
  * tested on `1.8.3` in a stand-alone Jupyter notebook
  * tested on upcoming `1.9.0` release in stand-alone Jupyter notebook
  * tested from `arcgispro-py3` environment shipped with `ArcGIS Pro 2.8` in stand-alone Jupyter notebook

**Comment by plarkin** [comment]
**Description:** 
This issue occurs after ArcGIS Server is improperly shut down. We experienced a major power outage yesterday and this error popped up. After restarting ArcGIS Server the issue persists. 

Ideally this issue would be reopened and tested under this scenario. [BUG-000136577](https://support.esri.com/en/bugs/nimbus/QlVHLTAwMDEzNjU3Nw==) report states that the alternate solution is to upgrade Pro to 2.7. This does not resolve the issue. Users in the Community forum are reporting this issue still exists on API 1.9.1. [post link](https://community.esri.com/t5/python-questions/attributeerror-server-object-has-no-attribute/td-p/1072637)

**To Reproduce**
Initiate a catastrophic shutdown of machine hosting ArcGIS Server. Restart machine and AGS. 

Run Code:
```
from arcgis.gis import server
site = server.Server(url,username="",password="",tokenurl="https://<server>.<host>:<port>/arcgis/rest/generateToken",expiration=60)
serverFolders = site.services.folders
```

Results in:

```
File "C:\Program Files\ArcGIS\Pro\bin\Python\envs\arcgispro-py3\lib\site-packages\arcgis\gis\server\_common\_base.py", line 81, in __getattr__
    return self._properties.__getitem__(name)
  File "C:\Program Files\ArcGIS\Pro\bin\Python\envs\arcgispro-py3\lib\site-packages\arcgis\_impl\common\_mixins.py", line 342, in __getitem__
    return self._mapping[key]
KeyError: 'services'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<path>\ServiceHealth.py", line 111, in <module>
    serverFolders = site.services.folders
  File "C:\Program Files\ArcGIS\Pro\bin\Python\envs\arcgispro-py3\lib\site-packages\arcgis\gis\server\_common\_base.py", line 86, in __getattr__
    raise AttributeError("'%s' object has no attribute '%s'" % (type(self).__name__, name))
AttributeError: 'Server' object has no attribute 'services'
```


**Expected behavior**
Normally services.folders returns a list of folders. 


**Platform** 
Code Run from: ArcGIS Pro 2.8.3 - Python API 1.8.5 , OS - VM Windows Server 2016 Standard (Build 1607) 
Code Target: ArcGIS Server 10.8.1, OS - Physical Windows Server 2016 Standard (Build 1607). ArcGIS Server is under a load balancer and proxy. Not federated.
